sections/Stake/TabContent/TabContent.tsx

```diff
@@ -0,0 +1,146 @@
+import { FC } from 'react';
+import styled from 'styled-components';
+import { wei } from '@synthetixio/wei';
+
+import Button from 'components/Button';
+import { FlexDivColCentered } from 'components/common';
+
+interface TabContentProps {
+	balance: any;
+	label: string;
+	action: string;
+	isApproved: boolean;
+	placeholder: string;
+	inputValue: number;
+	setIsMaxValue: Function;
+	setInputValue: Function;
```

Comment:
lets type all methods with more detail than just Function. So if might be (value: number) => void or (value: Wei) => void. Whatever the type is lets be more specific